python/test/sentencepiece_test.py

```diff
@@ -208,36 +198,31 @@ def test_ja_roundtrip(self):
 
   def test_train(self):
     spm.SentencePieceTrainer.Train(
-        '--input='
-        + os.path.join(data_dir, 'botchan.txt')
-        + ' --model_prefix=m --vocab_size=1000'
+        f'--input={BOTCHAN} --model_prefix=m --vocab_size=1000'
```

Comment:
Note: this introduces a requirement of Python >=3.6.
I could not find any official support policy.
[pypi stats](https://pypistats.org/packages/sentencepiece) indicate that there should not be a problem:

![image](https://github.com/user-attachments/assets/262ad9dd-f875-42e0-a7fe-063027e7cf1f)